cmd/app/flags.go

```diff
@@ -68,8 +68,8 @@ func configureFlags(command *cobra.Command) {
 		"When building a Hugo-compliant documentation bundle, files with filename matching one form this list (in that order) will be renamed to _index.md. Only useful with --hugo=true")
 	_ = vip.BindPFlag("hugo-section-files", command.Flags().Lookup("hugo-section-files"))
 
-	command.Flags().StringSlice("extracted-files-formats", []string{".md"},
-		"Supported content format extensions (exampel: .md)")
+	command.Flags().StringSlice("extracted-files-formats", []string{".md", ".yaml", ".html", ".xhtml", ".xml", ".txt", ".png"},
```

Comment:
```suggestion
	command.Flags().StringSlice("extracted-files-formats", []string{".md", ".yaml", ..., ".png"},
```
Let's add other common media files instead